### DISCUSSION TITLE
how to pass multiple props to a component from a markdown file

### DISCUSSION THREAD

**Discussion by cpt-n3mo** [nuxt/content#1492] [Q&A]
Good day..

how to pass multiple props to a component from a markdown file..?
as this is not documented and things like this do not work.
```
::comp{key1=val1,key2=val2}
::comp{[key1=val1,key2=val2]}
::comp{[{key1=val1},{key2=val2}]}
::comp{{key1=val1},{key2=val2}}
```

kind regards

**Comment by rensvis** [comment]
After some trial and error I figured it out. Separate your props with spaces instead of commas.

```
::component{imagePath="/path/to/image.jpg" imagePosition="left" stretchImage=true}
## 
::
```

Would be a great addition to the docs.
